api/test/e2e/label_test.go

```diff
@@ -200,6 +200,16 @@ func TestLabel(t *testing.T) {
 			ExpectStatus: http.StatusOK,
 			ExpectBody:   "{\"build\":\"16\"},{\"build\":\"17\"}",
 		},
+		{
+			Desc:         "get labels with the same key (key = build)",
+			Object:       ManagerApiExpect(t),
+			Method:       http.MethodGet,
+			Headers:      map[string]string{"Authorization": token},
+			Query:        "label=build:16,build:17",
+			Path:         "/apisix/admin/labels/all",
```

Comment:
🤔 What will happen if calling `/labels` only?

Reply:
It will return `{"error_msg":"404 Route Not Found"}`.